go/pkg/client/cas.go

```diff
@@ -101,7 +107,20 @@ func (c *Client) UploadIfMissing(ctx context.Context, data ...*chunker.Chunker)
 	log.V(2).Info("Waiting for remaining jobs")
 	err = eg.Wait()
 	log.V(2).Info("Done")
-	return err
+
+	result := &UploadIfMissingStats{
+		Hits:   make(map[digest.Digest]struct{}),
+		Misses: make(map[digest.Digest]struct{}),
+	}
+	for _, dg := range missing {
```

Comment:
maybe just returning `missing` is fine?

Reply:
Ack, a slice will be consistent with https://github.com/bazelbuild/remote-apis-sdks/blob/6f048255943e502b14630709f22412f23a7395c3/go/pkg/client/cas.go#L433. But I still feel like a `map` would be more convenient for the caller side. Anyway, if `map` makes more sense, maybe we should do that all together in another PR.